api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricHandler.java

```diff
@@ -89,752 +58,11 @@
-        Futures.addCallback(future, new NoDataCallback<>(asyncResponse));
-    }
-
-    @POST
-    @Path("/counters/{group}/{counter}")
-    @ApiOperation(value = "Increase value of a counter", hidden = true)
-    public void updateCounter(@Suspended final AsyncResponse asyncResponse, @PathParam("group") String group,
-        @PathParam("counter") String counter) {
-        ListenableFuture<Void> future = metricsService
-                .updateCounter(new Counter(DEFAULT_TENANT_ID, group, counter, 1L));
-        Futures.addCallback(future, new NoDataCallback<>(asyncResponse));
-    }
-
-    @POST
-    @Path("/counters/{group}/{counter}/{value}")
-    @ApiOperation(value = "Update value of a counter", hidden = true)
-    public void updateCounter(@Suspended final AsyncResponse asyncResponse, @PathParam("group") String group,
-        @PathParam("counter") String counter, @PathParam("value") Long value) {
-        ListenableFuture<Void> future = metricsService.updateCounter(new Counter(DEFAULT_TENANT_ID, group, counter,
-                value));
-        Futures.addCallback(future, new NoDataCallback<>(asyncResponse));
-    }
-
-    @GET
-    @Path("/counters/{group}")
-    @ApiOperation(value = "Retrieve a list of counter values in this group", hidden = true, response = Counter.class,
-            responseContainer = "List")
-    @Produces({ APPLICATION_JSON })
-    public void getCountersForGroup(@Suspended final AsyncResponse asyncResponse, @PathParam("group") String group) {
-        ListenableFuture<List<Counter>> future = metricsService.findCounters(group);
-        Futures.addCallback(future, new SimpleDataCallback<>(asyncResponse));
-    }
-
-    @GET
-    @Path("/counters/{group}/{counter}")
-    @ApiOperation(value = "Retrieve value of a counter", hidden = true, response = Counter.class,
-            responseContainer = "List")
-    public void getCounter(@Suspended final AsyncResponse asyncResponse, @PathParam("group") final String group,
-        @PathParam("counter") final String counter) {
-        ListenableFuture<List<Counter>> future = metricsService.findCounters(group, Collections.singletonList(counter));
-        Futures.addCallback(future, new FutureCallback<List<Counter>>() {
-            @Override
-            public void onSuccess(List<Counter> counters) {
-                if (counters.isEmpty()) {
-                    asyncResponse.resume(Response.status(404).entity("Counter[group: " + group + ", name: " +
-                            counter + "] not found").build());
-                } else {
-                    Response jaxrs = Response.ok(counters.get(0)).build();
-                    asyncResponse.resume(jaxrs);
-                }
-            }
-
-            @Override
-            public void onFailure(Throwable t) {
-                asyncResponse.resume(t);
-            }
-        });
-    }
-
-    @GET
-    @Path("/{tenantId}/metrics")
+    @Path("/{tenantId}")
```

Comment:
I think that the previous endpoint, `GET /{tenantId}/metrics`, makes more sense because there might be other information we want to include under `/{tenantId}`. There are tenant-level data retention and aggregation settings for example. I would not be surprised if introduce additional tenant-level configuration settings.